static/js/reducers/index.js

```diff
@@ -56,6 +61,45 @@ export const courseList = (state = INITIAL_COURSE_LIST_STATE, action) => {
   }
 };
 
+export const INITIAL_DIALOG_STATE = {
+  openDialog: false,
+  degreeLevel: '',
+  educationIndex: 0
+};
+export const educationDialog = (state = INITIAL_DIALOG_STATE, action) => {
+  switch (action.type) {
+    case SHOW_EDUCATION_FORM_DIALOG:
+      return Object.assign({}, state, {
+        openDialog: true,
+        degreeLevel: action.payload.level,
+        educationIndex: action.payload.index
+      });
+    case HIDE_EDUCATION_FORM_DIALOG:
+      return INITIAL_DIALOG_STATE;
+    default:
+      return state;
+  }
+};
+export const INITIAL_EDUCATION_LEVEL_STATE = {
+  hs: true,
+  a: true,
+  b: true,
+  m: false,
+  d: false
+};
+export const educationLevels = (state = INITIAL_EDUCATION_LEVEL_STATE, action) => {
+  switch (action.type) {
+    case TOGGLE_EDUCATION_LEVEL:
```

Comment:
Can you change this action to provide the new state as an argument? It will make it slightly easier to test the action.